ramls/mod-users-bl.raml

```diff
@@ -218,6 +219,43 @@ resourceTypes:
           body:
             text/plain:
               example: "Internal server error"
+  /login-with-expiry:
+    post:
+      description: |
+        Allow a new user to login and return two cookies, one containing the user's refresh token
+        and one containting an access token. Both tokens have an expiration. The expiration time
+        for each is contained in the composite user token expiration proerty.
+      is: [permissionsExpandable, includeable]
+      headers:
+        User-Agent:
+        X-Forwarded-For:
+      body:
+        application/json:
+          schema: loginCredentials
+      responses:
+        201:
+          body:
+            application/json:
+              schema: compositeUser
+          headers:
+            # TODO Remove since it wil be in non-raml set-cookie
+            # TODO May need to not respond with raml object in /login too so that code generation for other things doesn't break.
+            x-okapi-token:
```

Comment:
Can these TODOs be addressed during this pull request?

Reply:
This was an oversight on my part. I should have removed those TODOs and added a comment in the raml instead to explain that the two set-cookie headers are part of the response, but aren't visible in the RAML. There's no x-okapi-token header returned, only the two set-cookie ones.